compat/Makefile.am

```diff
@@ -93,7 +93,7 @@ TESTS += testHeaders
 ## Special Universal .h dependency test script
 ## aborts if error encountered
 testHeaders: $(srcdir)/*.h $(srcdir)/os/*.h
-	$(SHELL) $(top_srcdir)/test-suite/testheaders.sh "$(CXXCOMPILE)" $^ || exit 1
+	$(MAKE) $(^:.h=.hdrtest)
 
 CLEANFILES += testHeaders
```

Comment:
Please check if this can be dropped now that `testHeaders` is no longer a file object created by `testheaders.sh`. If it can all other `Makefile.am` and the shared `TestHeaders.am` should have the same removal.

Reply:
there needs to be some executable, so I've left that part in - a bit more generic and one in Makefiles so that it can be parallelised